checker/src/org/checkerframework/checker/i18nformatter/I18nFormatterTransfer.java

```diff
@@ -63,7 +63,8 @@ public I18nFormatterTransfer(I18nFormatterAnalysis analysis, I18nFormatterChecke
                     thenStore.insertValue(firstParam, anno);
                 }
             } catch (FlowExpressionParseException e) {
-                // errors are reported at declaration site
+                // report errors here
+                checker.report(e.getResult(), node);
```

Comment:
I think that would confuse the user. We are parsing a fixed string (`#1`) to retrieve an argument of a call to a method that was annotated with `@I18nChecksFormat` - there is no `@I18nFormatFor` annotation involved in this process.